hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -71,42 +107,50 @@ public int getZkfcHeapSize() {
   }
 
   public int getTaskHeapSize(String taskName) {
+    int size = DEFAULT_HEAP_SIZE;
```

Comment:
This variable DEFAULT_HEAP_SIZE is actually never used since we will throw a ConfigurationException in the default case. Either we should return 0 or just return inside the switch case statement as it was before